.github/workflows/update_dist.yml

```diff
@@ -1,19 +1,28 @@
+permissions:
+  id-token: write
```

Comment:
Just to make sure I understand this right, this is so ESC can authenticate using `OIDC`?  

Do we not need the `auth-actions` as described here? https://github.com/pulumi/esc-action?tab=readme-ov-file#full-example-using-pulumiauth-actions-for-authentication

It feels like some of this could benefit from better documentation for the action.

Reply:
Yes, definitely. Will get that updated.

Reply:
Done. The README for the action now documents all of this.